pyoptsparse/pyIPOPT/src/callback.c

```diff
@@ -34,7 +34,9 @@
 */
 
 #include "hook.h"
-#include <unistd.h>
+#ifndef _WIN64
+#include <unistd.h>  // is this needed? not platform independent, compiles successfully w/out
```

Comment:
This might be a leftover from older versions of IPOPT, we should test this and possibly tag a supported release.

Reply:
Oh this is the file I meant... not pyipoptcore. It compiles and passes on Linux without it but I don't know the code well enough to remove it.

Reply:
Yeah I'm not sure, but if all tests pass without it then I think we can get rid of it.